cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java

```diff
@@ -84,11 +84,14 @@ public static String triggerKeyForPartition(DatasetId datasetId) {
     return "partition:" + datasetId.getNamespace() + '.' + datasetId.getDataset();
   }
 
-  public static Set<String> triggerKeysForProgramStatus(ProgramId programId, Set<ProgramStatus> programStatuses) {
+  public static String triggerKeyForProgramStatus(ProgramId programId, ProgramStatus programStatus) {
```

Comment:
Should rebase now that this method was already added in the implement program status scheduling PR.